app/lib/actions/katello/content_view_version/export.rb

```diff
@@ -0,0 +1,70 @@
+module Actions
+  module Katello
+    module ContentViewVersion
+      class Export < Actions::EntryAction
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        ISO_OUTPUT_BASEDIR = "/var/lib/pulp/published/yum/master/group_export_distributor/"
+
+        def plan(content_view_version, export_to_iso, since, iso_size)
+          # assemble data to feed to Pulp
+          start_date = since ? since.iso8601 : nil
+          content_view = ::Katello::ContentView.find(content_view_version.content_view_id)
+          org_label = ::Organization.find_by(:id => content_view.organization_id).label
+          repo_pulp_ids = content_view.repositories.collect { |r| r.pulp_id }
```

Comment:
ugh, that would have been a tricky one to find if it wasn't caught during review. I will update now.